media-sound/gmusicbrowser/gmusicbrowser-1.1.15-r2.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit fdo-mime gnome2-utils
+
+DESCRIPTION="An open-source jukebox for large collections of mp3/ogg/flac files"
+HOMEPAGE="http://gmusicbrowser.org/"
+SRC_URI="http://${PN}.org/download/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dbus doc extras gstreamer gstreamer-0 libnotify mplayer"
+
+GSTREAMER_DEPEND="dev-perl/Glib-Object-Introspection"
+GSTREAMER0_DEPEND="
+	dev-perl/GStreamer
+	dev-perl/GStreamer-Interfaces
+	media-plugins/gst-plugins-meta:0.10"
+MPLAYER_DEPEND="media-video/mplayer"
+MPV_DEPEND="media-video/mpv"
+OTHER_DEPEND="
+	media-sound/alsa-utils
+	media-sound/flac123
+	|| ( media-sound/mpg123 media-sound/mpg321 )
+	media-sound/vorbis-tools"
+
+RDEPEND="dev-lang/perl
+	dev-perl/gtk2-perl
+	virtual/perl-MIME-Base64
+	|| ( net-misc/wget dev-perl/AnyEvent-HTTP )
+	dbus? ( dev-perl/Net-DBus )
+	gstreamer? ( ${GSTREAMER_DEPEND} )
+	gstreamer-0? ( ${GSTREAMER0_DEPEND} )
+	mplayer? ( || ( ${MPLAYER_DEPEND} ${MPV_DEPEND} ) )
+	!gstreamer? ( !gstreamer-0? ( !mplayer? ( ${OTHER_DEPEND} ) ) )
+	extras? ( dev-perl/gnome2-wnck )
+	libnotify? ( dev-perl/Gtk2-Notify )"
+DEPEND="sys-devel/gettext"
+
+src_install() {
+	emake \
+		DOCS="AUTHORS NEWS README" \
+		DESTDIR="${D}" \
+		iconsdir="${D}/usr/share/icons/hicolor" \
+		install
+
+	use doc && dohtml layout_doc.html
```

Comment:
`dohtml` is deprecated in `EAPI=6`. Instead, do the following:
```
use doc && local HTML_DOCS=( layout_doc.html )
einstalldocs
```
and then you can likely also nuke the `DOCS` argument from the `emake` command, as those files are part of the standard set.